### DISCUSSION TITLE
ABOUT YOUR 💩 2FA

### DISCUSSION THREAD

**Discussion by Manu-sh** [community/community#69359] [Code Security]
### 2FA Nobody asked for

Product Feedback

I don't like this invasion of privacy, you shouldn't have my number and **force** user to use your 💩 ""tech"" i don't wanna use, i know you OS is full of bugs and 💩, like telemetry and so on... but there is no such problems on GitHub, **have you introduced any new security issues lately dear M$**?

**Comment by AwokeKnowing** [comment]
I recommend everyone leave github

They are now just become bullies and another Microsoft app. 

They are unfit to host the open source community's work. Now they don't even let you decide how to authenticate. they force you to tie your account to SMS or some app. 

Move on.  

Github has lost it's way.Forcing 2fa against dev's will is EXACTLY the kind of behavior that made MS infamous. Why kill github? why don't you get it? MAKE 2FA OPTIONAL
